### PR TITLE
Changing default confidence threshold for new detectors from 0.9 to 0.75

### PR DESCRIPTION
**Pull request by timmarkhuff**
This PR changes default confidence threshold for new detectors from 0.9 to 0.75.

The rationale is:
1) the web app uses 0.75, so we should be consistent
2) 0.75 is a more reasonable default, and trying to get started with a detector that has a confidence threshold of 0.9 can be painful because there are so many escalations. 

I updated `public-api.yaml` and then ran `make generate` to update the necessary files, according to the instructions in `DEVELOPING.md`.

### CODE REVIEW

**Review by brandon-groundlight — APPROVED** [review]
I forgot, changing the public-api.yaml is the reason we didn't do this sooner. This is fine, but this is the Absolute Last time that anyone should manually edit the public-api.yaml file

**Review by robotrapta — APPROVED** [review]
This should bump us up to 0.16

**Comment by timmarkhuff** [comment]
> I forgot, changing the public-api.yaml is the reason we didn't do this sooner. This is fine, but this is the Absolute Last time that anyone should manually edit the public-api.yaml file

Okay, at some point you'll need to teach me what the alternative is :)

File: pyproject.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ packages = [
     {include = "**/*.py", from = "src"},
 ]
 readme = "README.md"
-version = "0.15.2"
+version = "0.15.3"
```
[the remark this answers is not in the document]
Comment (timmarkhuff) [reply pyproject.toml]: Done.

**Pull request merged**